app/utils/populations.py

```diff
@@ -0,0 +1,43 @@
+import requests
+from io import StringIO, BytesIO
+from cachetools import cached, TTLCache
+from zipfile import ZipFile, ZipInfo
+from .countrycodes import country_code
+
+# Fetching of the populations.
+def fetch_populations():
+    """
+    Returns a dictionary containing the population of each country fetched from the GeoNames (https://www.geonames.org/).
+    
+    :returns: The mapping of populations.
+    :rtype: dict
+    """
+    print ("Fetching populations...")
+
+    # Mapping of populations
+    mappings = {}
+
+    # Fetch the countries.
+    countries = requests.get("http://api.geonames.org/countryInfoJSON?username=dperic").json()['geonames']
+
+    # Go through all the countries and perform the mapping.
+    for country in countries:
+        mappings.update({ country["countryCode"]: int(country["population"]) or None })
+
+    # Finally, return the mappings.
+    return mappings
+
+# Mapping of alpha-2 codes country codes to population.
+populations = fetch_populations()
```

Comment:
You could make this a "start_up event"
https://fastapi.tiangolo.com/advanced/events/

Reply:
Hmm, it seems to work on startup now though? @Kilo59

Reply:
Absolutely, what you have here looks like it would work fine. Just pointing out a feature of FastAPI that can help achieve the same thing.

Reply:
@Kilo59 alright :) We can look at it later. Now populations should work all perfectly. That's a nice feature.